packages/xo-server-backup-reports/package.json

```diff
@@ -43,6 +43,7 @@
     "@babel/cli": "^7.0.0",
     "@babel/core": "^7.0.0",
     "@babel/preset-env": "^7.0.0",
+    "@babel/plugin-proposal-function-bind": "7.0.0",
```

Comment:
Avoid using this plugin/syntax in new code, because it will probably be deprecated in the future.